addons/website/static/src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,14 @@
+header {
+    .o_main_navbar {
```

Comment:
In general, I think that modules shouldn't target webclient classes. 
I'd use `.o_website_systray` as main parent selector instead.